docker/Dockerfile

```diff
@@ -0,0 +1,170 @@
+# Copyright 2019 Intel Corporation
```

Comment:
2019 -> 2020

Reply:
This is a old file renamed to Dockerfile in this PR. Hence update is not needed.

Reply:
Ok